### PR TITLE
Add home cards for vaccination status (EXPOSUREAPP-6730, EXPOSUREAPP-6731)

### PR DESCRIPTION
**Pull request by d4rken**
There is test code in `VaccinationStorage` you can alter to view the different cards.

* Card for (in)complete vaccination status
* Wired up to repository
* No click actions yet, needs follow up PR.
* ~~Vaccination will currently be an "ID" because we need to get the name from `VaccinationValueSets` which are not yet implemented.~~

- [x] Get fixed card graphics by our resident artist @jurajkusnier 

So this is about the layouts, the structure, and the wiring.

### CODE REVIEW

**Comment by d4rken** [comment]
From layout preview

Screenshot from 2021-05-05 17-28-17
Screenshot from 2021-05-05 17-27-56

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/VaccinatedPerson.kt [hunk 1]
```diff
@@ -26,12 +23,21 @@ data class VaccinatedPerson(
...
+    val vaccinationName: String
+        get() = vaccinationCertificates.first().vaccineName
+
     val firstName: String?
         get() = vaccinationCertificates.first().firstName
 
     val lastName: String
         get() = vaccinationCertificates.first().lastName
 
+    val combinedName: String
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/VaccinatedPerson.kt:38]: maybe `fullName` ?

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/VaccinatedPerson.kt [hunk 2]
```diff
@@ -26,12 +23,21 @@ data class VaccinatedPerson(
             it.toProofCertificate(valueSet)
         }.toSet()
 
+    val vaccinationStatus: Status
+        get() = if (proofCertificates.isNotEmpty()) Status.COMPLETE else Status.INCOMPLETE
+
+    val vaccinationName: String
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/VaccinatedPerson.kt:29]: `vaccineName` ?

File: Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/repository/storage/VaccinationStorage.kt [hunk 3]
```diff
@@ -57,6 +58,22 @@ class VaccinationStorage @Inject constructor(
             }
         }
 
+    init {
+        // TODO REMOVE when all is done
+        if (CWADebug.isDebugBuildOrMode && !CWADebug.isAUnitTest) {
+            personContainers = setOf(
+                PersonData(
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/java/de/rki/coronawarnapp/vaccination/core/repository/storage/VaccinationStorage.kt:65]: `VaccinatedPersonData` ? we could have some person data somewhere else such RAT profile  too

File: Corona-Warn-App/src/main/res/layout/vaccination_home_complete_card.xml [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/vaccination_compelete_gradient"
+    android:padding="@dimen/card_padding">
+
+
+    <androidx.appcompat.widget.AppCompatImageView
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/vaccination_home_complete_card.xml:11]: `ImageView` is enough  as the one below and let the inflater do the work for you

File: Corona-Warn-App/src/main/res/layout/vaccination_home_incomplete_card.xml [hunk 5]
```diff
@@ -0,0 +1,93 @@
...
+<androidx.constraintlayout.widget.ConstraintLayout xmlns:android="http://schemas.android.com/apk/res/android"
+    xmlns:app="http://schemas.android.com/apk/res-auto"
+    xmlns:tools="http://schemas.android.com/tools"
+    android:layout_width="match_parent"
+    android:layout_height="wrap_content"
+    android:background="@drawable/vaccination_incomplete"
+    android:padding="@dimen/card_padding">
+
+
+    <androidx.appcompat.widget.AppCompatImageView
```
Comment (mtwalli) [inline Corona-Warn-App/src/main/res/layout/vaccination_home_incomplete_card.xml:11]: same here

**Review by mtwalli — DISMISSED** [review]
LGMT

**Comment by d4rken** [comment]
Small change. While reviewing another PR i noticed that the `SARS-CoV-2 Impfschutz` is not dynamic but can be hardcoded.

**Review by mtwalli — DISMISSED** [review]

**Review by jurajkusnier — DISMISSED** [review]
LGTM

**Comment by d4rken** [comment]
Card graphics changed.

Screenshot from 2021-05-10 10-34-12
Screenshot from 2021-05-10 10-33-58
Screenshot from 2021-05-10 10-33-46

**Review by ChristinaTsigganos — APPROVED** [review]
UA approved

**Pull request merged**